libyul/CompilabilityChecker.cpp

```diff
@@ -48,8 +48,9 @@ map<YulString, int> CompilabilityChecker::run(
 
 		BuiltinContext builtinContext;
 		builtinContext.currentObject = &_object;
-		for (auto name: _object.dataNames())
-			builtinContext.subIDs[name] = 1;
+		for (auto const& subNode: _object.subObjects)
+			if (auto* subObject = dynamic_cast<Object*>(subNode.get()))
+				builtinContext.subIDs[subObject->name] = 1;
```

Comment:
Now data objects are no longer included in `subIDs`. Is that intentional?

Reply:
Fixed